module/zfs/vdev.c

```diff
@@ -1515,6 +1515,24 @@ vdev_metaslab_group_create(vdev_t *vd)
 	}
 }
 
+void
+vdev_update_nonallocating_space(vdev_t *vd, boolean_t add)
+{
+	spa_t *spa = vd->vdev_spa;
+	if (vd->vdev_mg->mg_class == spa_normal_class(spa)) {
```

Comment:
You could inverse the condition to remove following indentation.  Also while there, please add empty line after `spa` definition.